### ISSUE TITLE
python_linux_dbg_native.py27.test.unit.beta._face_interface_test.DynamicInvokerBlockingInvocationInlineServiceTest failing on Linux Master

### ISSUE THREAD

**Issue by dgquintas** [grpc/grpc#10049]
https://grpc-testing.appspot.com/job/gRPC_master_linux/1701/testReport/junit/(root)/python_linux_dbg_native/py27_test_unit_beta__face_interface_test_DynamicInvokerBlockingInvocationInlineServiceTest/

```
Errors/Failures: 
[91munit.beta._face_interface_test.DynamicInvokerBlockingInvocationInlineServiceTest.testSuccessfulUnaryRequestUnaryResponse[0m
[1mtraceback:[0m
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/case.py", line 329, in run
    testMethod()
  File "/var/local/git/grpc/src/python/grpcio_tests/tests/unit/framework/interfaces/face/_blocking_invocation_inline_service.py", line 94, in testSuccessfulUnaryRequestUnaryResponse
    request, test_constants.LONG_TIMEOUT, with_call=True)
  File "/var/local/git/grpc/py27/local/lib/python2.7/site-packages/grpc/beta/_client_adaptations.py", line 322, in __call__
    self._request_serializer, self._response_deserializer)
  File "/var/local/git/grpc/py27/local/lib/python2.7/site-packages/grpc/beta/_client_adaptations.py", line 208, in _blocking_unary_unary
    raise _abortion_error(rpc_error_call)
LocalError: LocalError(code=StatusCode.UNIMPLEMENTED, details="Method not found!")
```

**Comment by nathanielmanistaatgoogle** [comment]
P0? How reproducible?

**Comment by dgquintas** [comment]
P0 because it's happening on Master. It's certainly flaky.

**Comment by nathanielmanistaatgoogle** [comment]
How certain are you about "certainly"? Any idea of the flake rate? This issue smells like a duplicate of [issue 9435](https://github.com/grpc/grpc/issues/9435) and if it is then rather than two tests that each have different problems it means that there's some very-low-rate spurious `UNIMPLEMENTED`/"Method not found!" defect happening across at least all Python tests.

I'd like to know what is the method is that is not found, but I'm wary of echoing in the details string possibly-hostile content sent by the client as "the method". Maybe I'll write some sanitization; our tests only ever use methods containing letters, forward slashes, and periods (I think).

**Comment by dgquintas** [comment]
I'm only technically correct, in that it flaked (once): https://grpc-testing.appspot.com/job/gRPC_master_linux/1701/testReport/junit/(root)/python_linux_dbg_native/py27_test_unit_beta__face_interface_test_DynamicInvokerBlockingInvocationInlineServiceTest/history/

However undesirable that is, it seems to have happened only once in a while. I'm removing my zealous P0. Would it be possible to echo the missing method from the test, as opposed to modifying the library code, which as you say would be too revealing?

**Comment by nathanielmanistaatgoogle** [comment]
It wouldn't be possible because (1) the problem is server-side and almost all the tests control the system under test from the client side and (2) the next time the problem is observed (which could be weeks given the time between [issue 9435](https://github.com/grpc/grpc/issues/9435) and this issue, the only two reports) it could come up in _any_ test, not just either of the two tests in which it's been observed.

I think I'll add server-side sanitization. Sit tight.

**Comment by dgquintas** [comment]
:+1: thanks!

**Comment by dgquintas** [comment]
Not seen in quite a while, closing.

**Comment by markdroth** [comment]
Just saw this again:

https://sponge.corp.google.com/target?id=5664c3f3-315a-4dd2-8f6a-2fdceeb1f5f9&target=github/grpc/python_linux_opt_native&searchFor=&show=FAILED&sortBy=STATUS

**Comment by jtattermusch** [comment]
Not seen in a long time & test also seems removed by  https://github.com/grpc/grpc/pull/14702.
